src/swapd/runtime.rs

```diff
@@ -838,39 +907,27 @@ impl Runtime {
                     // whether we're Bob or Alice and that we're on a compatible state
                     Msg::MakerCommit(remote_commit)
                         if self.state.commit()
-                            && self.state.trade_role() == Some(TradeRole::Taker) =>
+                            && self.state.trade_role() == Some(TradeRole::Taker)
+                            && self.state.remote_commit().is_none() =>
                     {
-                        trace!("received commitment from counterparty, can now reveal");
-                        let next_state = self
-                            .state
-                            .clone()
-                            .sup_commit_to_reveal(remote_commit.clone());
-
-                        match self.state.swap_role() {
-                            SwapRole::Alice => {}
-                            SwapRole::Bob => {
-                                let addr = self
-                                    .state
-                                    .b_address()
-                                    .cloned()
-                                    .expect("address available at CommitB");
-                                let msg = format!(
```

Comment:
This message is no longer pushed to the progress report. Is your goal to put the funding address and amount under a different cli endpoint? Like it stands currently I cannot drive the swap with just the cli.

Reply:
ops, i guess i forgot to push

Reply:
@TheCharlatan can you report precisely the exact conditions you don't get the stuff pushed to the progress report?

Reply:
6d26a61 is the correct commit i tested

Reply:
No, you are right. Progress shows up as before. Maybe I forgot to pull, sorry!